ParallelRoadTool/UI/UIMainWindow.cs

```diff
@@ -22,14 +22,18 @@ public class UIMainWindow : UIPanel
             new SavedInt("toggleY", ParallelRoadTool.SettingsFileName, -1000, true);
 
         private UIOptionsPanel _mainPanel;
-        private UINetList _netList;
+        //private UINetList _netList;
+        public UINetList _netList;
```

Comment:
I'd keep this private and add utility methods to access it.

Reply:
Yeah, this is part of the import hack that I don't like.